### ISSUE TITLE
Documentation Out of Date

### ISSUE THREAD

**Issue by michaeltlombardi** [RamblingCookieMonster/PSDeploy#88]
The [official docs site](http://psdeploy.readthedocs.io/en/latest/) seems to be out of date. For example, while a [document for the PSGalleryScript deployment option](https://github.com/RamblingCookieMonster/PSDeploy/blob/master/docs/Example-PSGalleryScript-Deployment.md) was added a year ago, it does not exist in the published docs.

I see we're using RTD for docs hosting, perhaps the link to the project source is broken somehow?

**Comment by RamblingCookieMonster** [comment]
Good catch!  I think it might be due to files added to docs folder but not linked in the [yaml](https://github.com/RamblingCookieMonster/PSDeploy/blob/master/mkdocs.yml) - I'll try to remember to run through these to get them all in.

PSGalleryScript in particular might be... iffy.  I vaguely recall it not being quite complete and realizing I wasn't a huge fan of the PSGalleryScript, and then, `squirrel!` - might be why it was never added to the index

Will leave this open as a reminder, thanks!
